lib/logstash/inputs/graphite.rb

```diff
@@ -21,6 +21,8 @@ class LogStash::Inputs::Graphite < LogStash::Inputs::Tcp
   public
   def run(output_queue)
     @queue = output_queue
+    # pass self as output_queue to super Tcp#run - this is a hack so that the << calls in
+    # Tcp will actually call the << method defined below. This is twisted :P
     super(self)
   end
```

Comment:
This is beyond madness. created https://github.com/logstash-plugins/logstash-input-graphite/issues/6